lambdas/data-migration1/src/index.ts

```diff
@@ -58,8 +69,7 @@ export const migrateCollectionRecord = async (
     .where('version', dynamoRecord.version);
   // Skip record if it was already migrated.
```

Comment:
Nit:  In the context of the function it's not longer skipping.  Take or leave, shouldn't hold up PR.